src/model/include/iDynTree/Model/Indeces.h

```diff
@@ -8,30 +8,10 @@
 #ifndef IDYNTREE_INDECES_H
 #define IDYNTREE_INDECES_H
 
-#include <string>
+#ifdef __DEPRECATED
```

Comment:
Who defines the `__DEPRECATED` macro?

Reply:
The `_DEPRECATED` macro is never defined in MSVC, so those warnings are not printed at all in MSVC, so no need to suppress them!

Reply:
Sorry I thought it was the PR on disabling this warnings.